backend/wmg/pipeline/summary_cubes/expression_summary/job.py

```diff
@@ -20,13 +20,25 @@
 
 
 def _load(
-    uri: str, gene_ontology_term_ids: list, cube_index: pd.DataFrame, cube_sum: np.ndarray, cube_nnz: np.ndarray
+    uri: str,
+    gene_ontology_term_ids: list,
+    cube_index: pd.DataFrame,
+    cube_sum: np.ndarray,
+    cube_nnz: np.ndarray,
+    cube_sum_raw: np.ndarray,
+    cube_nnz_raw: np.ndarray,
 ) -> (list, dict):
     """
     Build expression summary cube in memory and write to disk
     """
     dims, vals = build_in_mem_cube(
-        gene_ontology_term_ids, cube_index, expression_summary_non_indexed_dims, cube_sum, cube_nnz
+        gene_ontology_term_ids,
+        cube_index,
+        expression_summary_non_indexed_dims,
+        cube_sum,
+        cube_nnz,
+        cube_sum_raw,
+        cube_nnz_raw,
```

Comment:
`*_raw` contains the original (before this PR) counts. `cube_sum/nnz` now contains the aggregate counts.